src/code/css/wasabee.css

```diff
@@ -589,14 +589,16 @@
 .wasabee-dialog-markeradd div.content {
   display: grid;
   grid-template-columns: repeat(2, auto);
+  row-gap: 4px;
+  column-gap: 4px;
```

Comment:
you can add `align-items: baseline;` to fix label/input vertical alignement

Reply:
Thanks for the tip 👍